jenkins/workers.py

```diff
@@ -0,0 +1,94 @@
+"""
+Use the jenkins queue and computer APIs to log info about workers.
+
+Usage:
+    `python jenkins/workers.py -j https://example.jenkins.com`
+
+Example Output:
+    [INFO] Counts of workers and queue length from Jenkins API:
```

Comment:
I may be missing something but I think the timestamp isn't necessary/desirable.
The intent is for this to be run via a jenkins job, and to output this info to the console. Then splunk will index the data. If there are timestamps in the output I'm not sure but I think that splunk might parse each line into a separate event, which we don't want.